docs/layouts/shortcodes/generated/high_availability_configuration.html

```diff
@@ -38,6 +38,12 @@
             <td>String</td>
             <td>Defines the ACL (open|creator) to be configured on ZK node. The configuration value can be set to “creator” if the ZooKeeper server configuration has the “authProvider” property mapped to use SASLAuthenticationProvider and the cluster is configured to run in secure mode (Kerberos).</td>
         </tr>
+        <tr>
+            <td><h5>high-availability.zookeeper.client.authorization</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>Map</td>
+            <td>Add connection authorization Subsequent calls to this method overwrite the prior calls.  In certain cases ZooKeeper requires additional Authorization information. For example list of valid names for ensemble in order to prevent the accidental connecting to a wrong ensemble.Each entry of type Map.Entry&lt;String, String&gt; will be transformed into an AuthInfo object with the constructor AuthInfo(String, byte[]). The field entry.key() will serve as the String scheme value, while the field entry.getValue() will be initially converted to a byte[] using the String#getBytes() method with UTF8 encodingIf not set the default configuration for a Curator would be applied.</td>
```

Comment:
```suggestion
            <td>Add connection authorization Subsequent calls to this method overwrite the prior calls. In certain cases ZooKeeper requires additional Authorization information. For example list of valid names for ensemble in order to prevent the accidental connecting to a wrong ensemble. Each entry of type Map.Entry&lt;String, String&gt; will be transformed into an AuthInfo object with the constructor AuthInfo(String, byte[]). The field entry.key() will serve as the String scheme value, while the field entry.getValue() will be initially converted to a byte[] using the String#getBytes() method with UTF8 encoding. If not set the default configuration for a Curator would be applied.</td>
```